draft-ietf-tls-subcerts.md

```diff
@@ -275,10 +275,10 @@ encode only the semantics that are needed for this application.
 struct {
   uint32 valid_time;
   opaque public_key<0..2^16-1>;
-} DelegatedCredentialParams;
+} Credential;
```

Comment:
Logically, this is an undelegated, or to-be-delegated, credential. The name "DelegatedCredentialParams" sounds like it's the parameters of the credential, e.g., valid_time.